modules/ROOT/pages/lb-create-cli.adoc

```diff
@@ -3,23 +3,30 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can create a dedicated load balancer using the Anypoint Platform CLI.
+You can create a dedicated load balancer (DLB) using the Anypoint Platform CLI.
 
 == Prerequisites
 
-To create a load balancer, ensure that you perform the following:
+To create a load balancer:
```

Comment:
"Before you create a load balancer:" or "Before you create a load balancer, you must have:"?

Because "to create a load balancer" is the next section, really.

These are prerequisites, that really should be worded, IMHO, as things that the user has already accomplished or conditions that already exist.